### PR TITLE
Fix prepend/extends graph tests

### LINKED ISSUE #2181: [BUG] Tests failing due to Chrome 104
Something funny is going on with async chunks in the latest version of Chrome. It's causing tests `grva004` and `grva005` (about extending/prepending data to traces) to fail. This is a fairly unusual edge case so the problems may be very narrow, but needs to be investigated ASAP to confirm how big an issue this is.

@T4rk1n suspects [Feature: Subresource loading with Web Bundles](https://chromestatus.com/feature/5710618575241216)

### PR DESCRIPTION
**Pull request by T4rk1n**
Replace graph prepend/extend trace test assertions with DOM nodes graph inspection. Resolve #2181

### CODE REVIEW

File: components/dash-core-components/tests/integration/graph/test_graph_varia.py [hunk 1]
```diff
@@ -193,6 +194,37 @@ def num_traces():
...
+    )
+
+    traces = browser.find_elements(f"{graph_id} .points")
+
+    for trace in traces:
+        elements = trace.find_elements(By.CSS_SELECTOR, "path")
+        for element in elements:
+            ActionChains(browser.driver).move_to_element(element).perform()
+            hover = browser.find_element(f"{graph_id} .hoverlayer")
+            content = hover.text
```
Comment (alexcjohnson) [inline components/dash-core-components/tests/integration/graph/test_graph_varia.py:210]: That'll work, though perhaps an easier solution (and more robust against changes in plotly.js implementation details) would be to write a little JS to look at `graphDiv._fullData[i].(x|y)`, where `graphDiv` is `document.querySelector("#{graph_id} .js-plotly-plot")`

**Comment by T4rk1n** [comment]
@alexcjohnson 
The thread killing doesn't work properly with flask 2.2.x, replaced the default runner with a multiprocess based runner. Not sure if we should replace `ProcessRunner` with the new one?

File: dash/testing/application_runners.py [hunk 2]
```diff
@@ -206,6 +207,59 @@ def stop(self):
...
+                app.run(threaded=True, **options)
+            except SystemExit:
+                logger.info("Server stopped")
+                raise
+            except Exception as error:
+                logger.exception(error)
+                raise error
+
+        # pylint: disable=import-outside-toplevel,no-member
+        import multiprocess as mp
```
Comment (alexcjohnson) [inline dash/testing/application_runners.py:235]: Why does this import need to be inside here?

Comment (T4rk1n) [reply dash/testing/application_runners.py]: Moved :ok_hand:

**Review by alexcjohnson — APPROVED** [review]
💃

File: components/dash-core-components/tests/integration/graph/test_graph_varia.py (hunk 1 above)
Comment (alexcjohnson) [reply components/dash-core-components/tests/integration/graph/test_graph_varia.py:210]: As discussed elsewhere, we can leave this for now, I may come back to it later myself.
